README.md

```diff
@@ -1,2 +1,25 @@
 # cmssw_vscode
 Recipies to develop CMSSW using VSCode
+
+## Indexing files
+Indexing files allows VSCode to easily resolve references to imported files.
+
+### C++
+Indexing C++ files is based on the *clangd* extension alongside a *compile-commands.json* file.
+1. Install clangd.
+    1. In VSCode in the side bar go to *Extensions*.
+    2. In the search bar type **clangd**.
+    3. Click *Install*.
+    4. Click *Install in SSH: lxplus.cern.ch*.
+    5. Disable IntelliSense if you have to installed (comes with the C/C++ Microsoft extension).
+        1. In VSCode go to *File -> Settings -> Remote [SSH: lxplus.cern.ch]*.
```

Comment:
I do not have File -> Settings option

<img width="1248" alt="Screenshot 2024-04-27 at 16 59 03" src="https://github.com/CTPPS/cmssw_vscode/assets/7374733/09bd4ad4-01a8-451b-b05e-af9a716c2246">

Reply:
I have them here
<img width="1055" alt="Screenshot 2024-04-27 at 17 00 00" src="https://github.com/CTPPS/cmssw_vscode/assets/7374733/8061475e-8fbc-4775-84e5-a88302e1935a">

Reply:
Interestingly enough, the [VSCode documentation](https://code.visualstudio.com/docs/getstarted/settings#_settings-editor) points to an even different route.

I changed it to use the Command Palette, so it should be more platform agnostic.